monitor.cpp

```diff
@@ -3,36 +3,43 @@
 #include <thread>
 #include <chrono>
 #include <iostream>
-using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+#include <string>
+using std::cout, std::endl, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
 
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if (temperature > 102 || temperature < 95) {
-    cout << "Temperature is critical!\n";
-    for (int i = 0; i < 6; i++) {
-      cout << "\r* " << flush;
-      sleep_for(seconds(1));
-      cout << "\r *" << flush;
-      sleep_for(seconds(1));
-    }
+void PrintMessage(const std::string& message) {
+  cout << message << endl;
+  for (int i = 0; i < 6; i++) {
+    cout << "\r* " << flush;
+    sleep_for(seconds(1));
+    cout << "\r *" << flush;
+    sleep_for(seconds(1));
+  }
+}
+
+int CheckBodyTemperature(float bodyTemperature) {
```

Comment:
This function returns an int, but the caller treats it as a boolean. Can make it return `bool` to make that explicit.
Also, name bool-return functions with "is". For example, `isTemperatureInRange`. Otherwise, `CheckBodyTemperature` can mean "return true when temperature is in range" or "return true when attention is needed (out of range)